src/github.com/getlantern/tarfs/tarfs.go

```diff
@@ -111,6 +111,12 @@ func (fs *FileSystem) SubDir(dir string) *FileSystem {
 	}
 }
 
+// GetIgnoreEmpty is the same as Get, but it ignores local file system files
+// if they're empty.
+func (fs *FileSystem) GetIgnoreEmpty(p string) ([]byte, error) {
```

Comment:
The function name is a bit misleading. Can we make the name more explicit, say, `GetIgnoreLocalEmpty`, or tweak `get` a bit to also apply `ignoreempty` flag to files in-memory?